app/facades/food_facade.rb

```diff
@@ -0,0 +1,15 @@
+class FoodFacade
+  class << self
+    def food_search(food)
+      data = FoodService.find_foods(food)
+      data[:foods].take(10).map do |food_data|
+        Food.new(food_data)
+      end
+    end
```

Comment:
Since the `result_count` method is hitting the same endpoint as the `food_search` method, we can actually consolidate this a bit and make our requests more efficient by taking all the info we need the first time. We could do something like this and remove the `result_count` method:

```ruby
  def food_search(food)
      data = FoodService.find_foods(food)
      foods = data[:foods].take(10).map do |food_data|
        Food.new(food_data)
      end
      [data[:totalHits], foods]
    end
```

Then in the controller, you can do:
`@result_count, @foods = FoodFacade.food_search(params[:q])`